### PR TITLE
Disable logging by prefixing a line

### PR DESCRIPTION
**Pull request by LeoVerto**
Implementation of [this](https://botbot.me/freenode/lincolnloop/2015-07-29/?msg=45842227&page=1) suggestion with some more configurability.

### CODE REVIEW

**Comment by LeoVerto** [comment]
Still trying to work out how to develop and test this repository.

**Comment by LeoVerto** [comment]
Okay, this should work, but unfortunately botbot-plugin's DummyLine is lacking some variables used here, so those have to be added first.

File: botbot/apps/plugins/tests/test_logger.py [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+    def setUp(self):
+        self.app = DummyApp(test_plugin=logger.Plugin())
+
+    def test_log(self):
+        log = self.app.respond("insert message here")
+        print log
+        self.assertEquals(log['text'], "insert message here", log)
+
+    def test_ignore(self):
+        log = self.app.respond("[off] test message, please ignore")
```
Comment (Freso) [inline botbot/apps/plugins/tests/test_logger.py]: Where are you setting the string to be "off" though? I don't even see "off" being given as the default in botbot/apps/plugins/core/logger.py.

Also, could you add a test for /me messages? To make sure it catches e.g. `/me [off] does something off-the-record`.

**Comment by LeoVerto** [comment]
Scrapped the testing due to the lack of a complete existing framework.

This feature is currently implemented on http://chatlogs.metabrainz.org/ and has survived extensive testing.

**Pull request closed without merging**